diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index be9df86be09c7655787a65c55ae6da01858894c3..356f2a045b0a6f1a0490f9a1a801df9f54163872 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -261,3 +261,8 @@ function expand_(str, max, maxLength, isTop) {
     return acc;
 }
 //# sourceMappingURL=index.js.map
+module.exports = Object.assign(exports.expand, {
+    expand: exports.expand,
+    EXPANSION_MAX: exports.EXPANSION_MAX,
+    EXPANSION_MAX_LENGTH: exports.EXPANSION_MAX_LENGTH,
+});
